src/main/webapp/authorization.js

```diff
@@ -0,0 +1,44 @@
+// GCP Generated Github OAuth configuration
+const config = {
+  apiKey: 'AIzaSyAR88Giah8cCEAvT_zDSIREWvgIIAeS8yY',
+  authDomain: 'step2020-279820.firebaseapp.com',
+};
+
+/**
+ * Initialize firebase if firebase is not already initialized. This method can
+ * be called as many times as we need without any side effects.
+ */
+function initializeFirebase() {
+  if (firebase.apps.length === 0) {
```

Comment:
Let's fix the lint error with `firebase`.

Like we discussed briefly in the sync, can we create a `getFirebase()` that just returns `firebase`? And only use `getFirebase`. That way, we can disable the lint error just within `getFirebase` and it could be much friendlier for testing.